Test/docker-compose.yml

```
@@ -61,14 +61,38 @@ services:
       target: dep-runner
     env_file:
       - ../Config/server.sample.env
+    # expose:
+    #   - "54000"
     ports:
-      - "50020:54000"
+      - "54000:54000"
     volumes:
       - type: bind
         source: ./../Config
         target: /QuickMPC-BTS/Config
+    # depends_on:
+    #   bts_envoy:
+    #     condition: service_started
```

I think it would be better to put it all together in this lines.

```suggestion
    ports:
      - "54000:54000"
    # expose:
    #   - "54000"
    # depends_on:
    #   bts_envoy:
    #     condition: service_started
```

---

@arukuka 
I fixed it with [2dff0b7](https://github.com/acompany-develop/QuickMPC-BTS/pull/7/commits/2dff0b749b0f21564f519271530edbe7f59adef4)